src/DataFactory/Bls12381DataFactory.js

```diff
@@ -2,10 +2,7 @@ import FateBls12381Fr from '../types/FateBls12381Fr.js'
 import FateBls12381Fp from '../types/FateBls12381Fp.js'
 import BaseDataFactory from './BaseDataFactory.js'
 
-const TYPES = [
-    'bls12_381.fr',
-    'bls12_381.fp',
-]
+const TYPES = ['bls12_381.fr', 'bls12_381.fp']
```

Comment:
I definitely prefer multiline list both for readability and maintenance regardless if it fits on line limit

Reply:
However, single line should be also allowed as it works better in some code blocks/snippets

Reply:
Prettier keep line breaks but only in objects: https://prettier.io/docs/rationale#multi-line-objects
The are only workarounds like `prettier-ignore` or adding an empty comment 🙁 
```js
// prettier-ignore
const TYPES = [ //
  'bls12_381.fr',
  'bls12_381.fp'
]
```